public/app/features/datasources/components/DataSourceConfigAlert.tsx

```diff
@@ -0,0 +1,222 @@
+import { css, cx } from '@emotion/css';
+import React, { AriaRole, HTMLAttributes, ReactNode } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { selectors } from '@grafana/e2e-selectors';
+import { useTheme2, IconName, Button, Icon, IconButton, AlertVariant, Link } from '@grafana/ui';
+
+export interface Props extends HTMLAttributes<HTMLDivElement> {
+  title: string;
+  /** On click handler for alert button, mostly used for dismissing the alert */
+  onRemove?: (event: React.MouseEvent) => void;
+  severity?: AlertVariant;
+  children?: ReactNode;
+  elevated?: boolean;
+  buttonContent?: React.ReactNode | string;
+  bottomSpacing?: number;
+  topSpacing?: number;
+  successMessage?: ReactNode;
+  exploreUrl: string;
+  canExploreDataSources: boolean;
+  dataSourceId: string;
+  onDashboardLinkClicked: () => void;
+}
+
```

Comment:
Why duplicate this whole component?

Reply:
My idea was to not add more complexity to the existing one, but I can just merge them